site/content/docs/10-exporting.md

```diff
@@ -42,9 +42,9 @@ You can also add a script to your package.json...
 
 ### How it works
 
-When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>` elements it finds, and captures any data served by the app.
+When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>`, `<img>` and `<source>` elements it finds, and captures any data served by the app.
 
-Because of this, any pages you want to be included in the exported site must either be reachable by `<a>` elements or added to the `--entry` option of the `sapper export` command.
+Because of this, any pages you want to be included in the exported site must either be reachable by `<a>`, `<img>` or `<source>` elements or added to the `--entry` option of the `sapper export` command.
```

Comment:
this sentence starts off talking about "pages", so maybe we shouldn't add `<img> or <source>` here since you probably wouldn't add them to `--entry`? (because what good is a directly crawled image if it's not reachable from some page)

Reply:
That's a good point. Do you think we need any changes to this sentence? I'm thinking maybe the addition to the previous sentence is enough.

Reply:
yeah, I think the addition to the previous sentence is probably enough